### DISCUSSION TITLE
Read playback state when streaming http web radio

### DISCUSSION THREAD

**Discussion by hansipete** [MusicPlayerDaemon/MPD#1367] [Q&A]
Hello there,

working a Raspberry Pi based radio player and would like to give feedback to the user if there's a problem with the stream: no/slow connected, refilling the buffer, etc.

I could not find a way to "read" this from mpd/mpc. Instead I'm polling `/proc/asound/card1/pcm0p/sub0/status` to check if there's audio signal... this works but i'd like to make this more solid.

Is there a way that I overlooked?

Thank you,
Hans

**Comment by dcabri** [comment]
That's easy:

mpc status
NPO Radio2: Harry Styles - As it was
[playing] #1/2  15:33/0:00 (0%)
volume: n/a   repeat: off   random: off   single: off   consume: off
